fix(SAML SSO): correct and improve configuration instructions

ugaston opened the pull request:
Versions: 7.5, 7.6, 7.7, 7.8

Try to make the configuration of the keycloak-saml.xml a bit clearer, especially for the cases where signature/encryption is required.

ugaston committed:
fix(SAML SSO): correct and improve configuration instructions

Versions: 7.5, 7.6, 7.7, 7.8

Try to make the configuration of the keycloak-saml.xml a bit clearer, especially for the cases where signature/encryption is required.

ugaston committed:
Update single-sign-on-with-saml.md

Trying to get the markdown formatting right....

ugaston committed:
fix(SAML SSO): Fix formatting and add troubleshooting info.

- Pass the information notes to the bottom of the bullet list in order to have a good formatting of the list.
- Add some common error examples and solution hints in the troubleshooting section.

abirembaut commented:
Awesome !
Good job !
Just a comment: I don't think we need to display the full stacktrace in the troubleshooting section as it takes a lot of space. The first 2 lines of the trace and the "caused by" line are clear enough to troubleshoot

I will make another PR to update the [keycloak-saml.xml](https://github.com/bonitasoft/bonita-engine-sp/blob/master/subscription/platform/platform-resources-sp/src/main/resources/tenant_template_portal/keycloak-saml.xml) accordingly

ugaston committed:
fix(SAML SSO): Shorten length of troubleshooting stack-traces

Take Anthony's review comments into account, and trim the troubleshooting stack-traces down to the first two lines and the 'caused by'.

abirembaut reviewed (approved):
LGTM

ugaston commented:
Cheers, Anthony!
